packages/playwright-core/src/remote/playwrightServer.ts

```diff
@@ -58,11 +59,10 @@ export class PlaywrightServer {
       assert(options.preLaunchedBrowser);
       this._preLaunchedPlaywright = options.preLaunchedBrowser.options.rootSdkObject as Playwright;
     }
-    if (mode === 'reuse-browser')
-      this._preLaunchedPlaywright = createPlaywright('javascript');
   }
 
-  preLaunchedPlaywright(): Playwright | null {
+  preLaunchedPlaywright(): Playwright {
+    this._preLaunchedPlaywright = createPlaywright('javascript');
```

Comment:
Should this method be re-entrant instead of creating a new playwright every time?